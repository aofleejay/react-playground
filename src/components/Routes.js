import React from 'react';
import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
import Feed from '../../src/containers/Feed';
import Profile from './Profile';

const Routes = () => (
  <BrowserRouter>
    <Switch>
      <Route exact path="/" component={Feed} />
      <Route exact path="/me" component={Profile} />
      <Redirect to="/" />
    </Switch>
  </BrowserRouter>
);

export default Routes;
