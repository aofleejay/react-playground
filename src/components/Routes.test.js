import React from 'react';
import { shallow } from 'enzyme';
import Routes from './Routes';

describe('Test Routes component', () => {
  it('Match its snapshot', () => {
    const wrapper = shallow(<Routes />);

    expect(wrapper).toMatchSnapshot();
  });
});
