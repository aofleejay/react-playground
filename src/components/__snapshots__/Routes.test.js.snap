// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Routes component Match its snapshot 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/me"
    />
    <Redirect
      push={false}
      to="/"
    />
  </Switch>
</BrowserRouter>
`;
